pkg/microservice/aslan/core/code/client/gerrit/gerrit.go

```diff
@@ -0,0 +1,83 @@
+/*
+Copyright 2022 The KodeRover Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package gerrit
+
+import (
+	"github.com/koderover/zadig/pkg/microservice/aslan/config"
```

Comment:
Adjust the import order of dependent packages

Reply:
fixed